test/e2e/provisioning/all_e2e_test.go

```diff
@@ -41,6 +42,21 @@ func TestInvalidObjectsGetRejected(t *testing.T) {
 	}
 }
 
+func TestKubevirtProvisioningE2E(t *testing.T) {
+	t.Parallel()
+
+	kubevirtKubeconfig := os.Getenv("KUBEVIRT_E2E_TESTS_KUBECONFIG")
+	if kubevirtKubeconfig == "" {
+		t.Fatalf("Unable to run kubevirt tests, KUBEVIRT_E2E_TESTS_KUBECONFIG must be set")
+	}
+
+	// Provisioning coreos images via kubevirt does not work, needs investigation
```

Comment:
Please create an issue so we don't forget about it + mention it in the docs

Reply:
https://github.com/kubermatic/machine-controller/issues/422